gtfs_realtime_translators/factories/factories.py

```diff
@@ -21,15 +21,31 @@ def __get_stop_time_events(arrival_time, departure_time=None):
 
         return arrival, departure
 
+    @staticmethod
+    def __set_delay_stop_time_events(arrival_delay, departure_delay):
```

Comment:
Something worth noting here 
When you find yourself with a _prefix_ or _suffix_ in a method name, there's a sign that you have another level of abstraction trying to get out.

In this case it appears there needs to be a model around _Stop Time Events_ and they appear polymorphic (delay, normal)